pkg/cache/unavailableofferings.go

```diff
@@ -48,13 +48,17 @@ func NewUnavailableOfferings() *UnavailableOfferings {
 }
 
 // IsUnavailable returns true if the offering appears in the cache
-func (u *UnavailableOfferings) IsUnavailable(instanceType ec2types.InstanceType, zone, capacityType string) bool {
+func (u *UnavailableOfferings) IsUnavailable(instanceType string, zone, capacityType string) bool {
```

Comment:
Why did we decide to change this from `ec2types.InstanceType`?

Reply:
Vestigial from a previous iteration, I can revert this change.

Reply:
Seems like it's still there?

Reply:
Shouldn't be anymore